test/unit/cute/volta/vectorization_auto.cu

```diff
@@ -109,7 +109,7 @@ template <class T, class GmemLayout, class RmemTiler>
 void
 test_copy_vectorization(GmemLayout gmem_layout, RmemTiler rmem_tiler)
 {
-  test_copy_vectorization<T>(DefaultCopy{}, gmem_layout, rmem_tiler);
+  test_copy_vectorization<T>(AutoVectorizingCopyWithAssumedAlignment<128>{}, gmem_layout, rmem_tiler);
```

Comment:
Since the DefaultCopy used to be `AutoVectorizingCopyWithAssumedAlignment<128>`, it must now be changed to explicitly specify this to preserve behavior. I believe this is the easiest way to reduce the scope of this PR and keep changes minimal.